packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -43,6 +43,8 @@ export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivEle
   isVertical?: boolean;
   /** Enables no border bottom tab styling */
   hasBorderBottom?: boolean;
+  /** Enables border bottom styling for secondary tabs */
+  hasSecondaryBorderBottom?: boolean;
```

Comment:
Is a new prop necessary? if the consumer specifies isSecondary and hasBorderBottom, could it do the same thing?

Reply:
Maybe we can update this with the next breaking change release? Assuming we want secondary tabs to also default to having a bottom border like default tabs.

Reply:
@kmcfaul I think that would be ideal.